pkg/rpc/client/rpc_test.go

```diff
@@ -716,16 +716,19 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 		{
 			name: "positive",
 			invoke: func(c *Client) (interface{}, error) {
-				return nil, c.SubmitBlock(block.Block{
+				return c.SubmitBlock(block.Block{
 					Base:         block.Base{},
 					Transactions: nil,
 					Trimmed:      false,
 				})
 			},
-			serverResponse: `{"jsonrpc":"2.0","id":1,"result":true}`,
+			serverResponse: `{"jsonrpc":"2.0","id":1,"result":{"hash":"0x72159b0cf1221110daad6e1df6ef4ff03012173b63c86910bd7134deb659c875"}}`,
```

Comment:
It doesn't differ from the tx hash from the previous commit. Strange.

Reply:
That was the wrong hash, replaced it with the actual one.

Reply:
I was surprised, that tests were still passed. Or not?

Reply:
Actually, it doesn't matter what hash is there, for proper test passing it should be just some uint256. These RPC client tests do not send requests to the real server, they use a stub which respond with specified `serverResonce`.